script/GameObjects/Elements/Material/RigidBody.lua

```diff
@@ -191,12 +191,12 @@ end
 
 function Entity:getToLocalMatrix()
     assert(self.body)
-    return self.body:getToLocalMatrix()
+    return self.body:getToLocalMatrix(Position.Identity())
 end
 
 function Entity:getToWorldMatrix()
     assert(self.body)
-    return self.body:getToWorldMatrix()
+    return self.body:getToWorldMatrix(Position.Identity())
```

Comment:
Here's the real meat. For now this doesn't do anything (the 64-bit positions are just converted to 32-bit ones), but in a future PR I will be passing the camera position here, and adjusting the view matrix to compensate for it.